### ISSUE TITLE
Release the Batch Category Picker without "Add all"

### ISSUE THREAD

**Issue by garethrees** [mysociety/alaveteli#6862]
The Category UI is actually quite good now.

We should re-scope the feature flag so that the Category UI itself is available to all pros, but the "Add all" option is still a beta/invite-only feature, as that's the thing we're worried about people mis-using.

We could try to fix https://github.com/mysociety/alaveteli-professional/issues/467 before we do this, but not essential.

Requires https://github.com/mysociety/alaveteli-professional/issues/567.

Fixes from https://github.com/mysociety/alaveteli-professional/issues/224.

Manually transferred from https://github.com/mysociety/alaveteli-professional/issues/568.

![Screenshot 2022-06-24 at 12 35 13](https://user-images.githubusercontent.com/282788/175526642-3b4b2c70-da1a-4b83-9130-17158a87fdd1.png)

**Comment by garethrees** [comment]
Can currently be enabled (with "add all") through the console:

```ruby
user = User.find_by(email: 'bob@example.com')
AlaveteliFeatures.backend.enable_actor(:pro_batch_category_ui, user)
```

Probably want to make "Add all" an on-request addition https://github.com/mysociety/alaveteli/issues/7021.

**Comment by gbp** [comment]
Released in 0.41.1
